lib/membrane/rtp/session_bin.ex

```diff
@@ -20,6 +20,18 @@ defmodule Membrane.RTP.SessionBin do
   `membrane_rtp_X_plugin` packages, where X stands for codec name. It's enough when such plugin is added to
   dependencies.
 
+  #### Jitter Buffer
+  Some depayloaders may need the incoming packets to be in a correct order therefore one needs to not only
+  use depayloader but also include a jitter buffer. It can be specified via pad's option `use_jitter_buffer?` when
```

Comment:
Yup, I refactored it so that we don't need the jitter buffer without the depayloader (or maybe you could find a usecase where we would use one without the other?). 

Have to update this documentation a little bit.